### ISSUE TITLE
twisted.web._newclient._WrapperException makes it a little tedious to debug

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4659]
|[<img alt="djfroofy's avatar" src="https://avatars.githubusercontent.com/u/89760?s=50" width="50" height="50">](https://github.com/djfroofy)| @djfroofy reported|
|-|-|
|Trac ID|trac#4659|
|Type|enhancement|
|Created|2010-09-30 21:35:01Z|

Getting a traceback (e.g. reason.printTraceback) with _WrapperException leads to something inscrutable. Example:
```
Unhandled Error
        Traceback (most recent call last):
        Failure: twisted.web._newclient.RequestGenerationFailed: [<twisted.python.failure.Failure <type 'exceptions.AttributeError'>>]
```
I don't know if it's possible but it would be nice to somehow get a meaningful traceback without having to introspect the exception type and doing stuff like:
```
for reason in reason.value.reasons:
    reason.printTraceback()
```

<details><summary>Searchable metadata</summary>

```
trac-id__4659 4659
type__enhancement enhancement
reporter__djfroofy djfroofy
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__web web
keywords__ 
time__1285882501000000 1285882501000000
changetime__1365601724000000 1365601724000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph commented|
|-|-|

(fixing formatting)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph commented|
|-|-|

[#5345](https://github.com/twisted/twisted/issues/5345) is related to this.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve set status to `closed`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve commented|
|-|-|

Closed in favor of [#5310](https://github.com/twisted/twisted/issues/5310).
